runner/lib/tasks/testnet.js

```diff
@@ -338,7 +339,7 @@ ${chainName} chain does not yet know of address ${soloAddr}
 
     const chainEnv = Object.create(process.env);
     chainEnv.SLOGFILE = slogFifo.path;
-    // chainEnv.DEBUG = 'agoric';
```

Comment:
Actually, this line should work, and give you more logs than VerboseDebugEnv.  The only reason you weren't seeing `swingset running` was that the timeout was too short.